python/global_scheduler/test/test.py

```diff
@@ -177,8 +176,6 @@ def test_integration_single_task(self):
                      2 * NUM_CLUSTER_NODES + 1)
 
     num_return_vals = [0, 1, 2, 3, 5, 10]
-    # There should not be anything else in Redis yet.
-    self.assertEqual(len(self.redis_client.keys("*")), 2 * NUM_CLUSTER_NODES + 1)
```

Comment:
Now that we are starting redis through `services.start_redis(cleanup=False)`, the `start_redis` command in `services.py` actually puts a key in the redis server saying what the start time of the process was (and we could imagine adding more things like a version number of something).